src/govuk/components/character-count/character-count.mjs

```diff
@@ -5,6 +5,42 @@ import '../../vendor/polyfills/Element/prototype/classList.mjs'
 import { closestAttributeValue, extractConfigByNamespace, mergeConfigs, normaliseDataset } from '../../common.mjs'
 import { I18n } from '../../i18n.mjs'
 
+/**
+ * @typedef {object} CharacterCountTranslations
```

Comment:
I've been keeping extra JSDoc comments at the bottom:

https://github.com/alphagov/govuk-frontend/blob/eslint-jsdoc/lib/file-helper.js#L151
https://github.com/alphagov/govuk-frontend/blob/eslint-jsdoc/src/govuk/common.mjs#L218

Do we want to decide if "at the top" is preferred?